pkg/arm/v5/resources.go

```diff
@@ -215,12 +241,98 @@ func (g *simpleGenerator) lbKubernetes() *network.LoadBalancer {
 					},
 					Name: to.StringPtr(lbKubernetesOutboundFrontendConfigurationName),
 				},
+				{
+					FrontendIPConfigurationPropertiesFormat: &network.FrontendIPConfigurationPropertiesFormat{
+						PrivateIPAllocationMethod: network.Dynamic,
+						PublicIPAddress: &network.PublicIPAddress{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/publicIPAddresses",
+								ipInboundName,
+							)),
+						},
+					},
+					Name: to.StringPtr(lbKubernetesInboundFrontendConfigurationName),
+				},
 			},
 			BackendAddressPools: &[]network.BackendAddressPool{
 				{
 					Name: to.StringPtr(lbKubernetesBackendPoolName),
 				},
 			},
+			LoadBalancingRules: &[]network.LoadBalancingRule{
+				{
+					LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
+						FrontendIPConfiguration: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/frontendIPConfigurations/" + lbKubernetesInboundFrontendConfigurationName),
```

Comment:
I wonder if it's worth using path.Join() here, in case paths elements are not clean. It seems like everything is constant, static, or generated correctly but just a thought.